src/core/Microsoft.PowerToys.Settings.UI/Views/ShortcutGuidePage.xaml

```diff
@@ -124,7 +124,7 @@
                     HorizontalAlignment="Left"
                     Margin="{StaticResource SmallTopBottomMargin}"
                     RelativePanel.Below="DescriptionPanel">
-                <Image Source="ms-appx:///Assets/Modules/ShortcutGuide.png" />
+                <Image Source="https://aka.ms/powerToysShortcutGuideSettingImage" />
```

Comment:
Your now downloading an image which could fail.  How does this look in that case?

Reply:
sure, these should be reverted.